src/control.c

```diff
@@ -338,19 +338,17 @@ void motor_control(void)
 	left_speed = get_encoder_left_speed();
 	right_speed = get_encoder_right_speed();
 	encoder_feedback_linear = (left_speed + right_speed) / 2.;
-	encoder_feedback_angular = get_encoder_angular_speed();
+	encoder_feedback_angular = -get_gyro_z_radps();
```

Comment:
Change the name of the variable. The angular feedback is not coming from the encoder now.